src/model/AssetsCompiler.ts

```diff
@@ -107,10 +109,25 @@ export class AssetsCompiler {
         private readonly runConfig: IAssetsCompilerConfiguration,
         private readonly repositoryDir: string
     ) {
+        if (AssetsCompiler.shouldUseAscLocal()) {
+            console.warn(
```

Comment:
it could make sense to throw an error and exit. Do we want to enable developers to continue using old cplace-asc?

Reply:
As discussed in the daily. It will give them some transition time